src/miner.app.src

```diff
@@ -26,9 +26,11 @@
     xmerl,  %% nat_upnp has a missing dep
     jsx,
     kvc,
-    longfi
+    longfi,
```

Comment:
I think that these should be sibyl deps and thus started automatically?

Reply:
yeah, these are unnecessary.  will remove